### DISCUSSION TITLE
Notes - 22 March 2024

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#24] [General]
# Update to GDP Lecture

- no major updates

# Lecture Reviews

## Distributions and Probability

- Bernoulli is a distribution for only two outcomes - make that more clear
- Economic examples for the distributions(?) (Poisson, for example, - useful for calculating the odds of a certain event, or a link to another lecture where it is used / wikipedia)
- Comparison between distributions
- Use case in economics and examples
- We could have the empirical distributions first  and then go on to the theoretical distributions (switch the order of the lectures)
- **Idea:** summary table at the end (mean, variance, cdf and pdf of each one)
- label x,y axis in figures
- add mystnb figures wrappers

## LLN and CLT

- update the code `X = st.bernoulli.rvs(p)` to show they are draws (so we should generate the draws, 10 of them) and link to wikipedia article around independent/quasi-independent. 
- consider promoting the theory at the end `17.2.1` above the simulations(?) -- could make it clear and use the code as confirmation. But does a good job of providing intuition.  Provide some context around Equation 17.1
- In 17.2.4 -- improve clarity around the loop 1 to 3 (sample of sample means) -- less vague language (use algorithm environment from sphinx-proof)
- In 17.2.4 remove the plot of the histogram in section (keep the violin plots and refer to these plots from the CLT section)
- In section 17.2.4 perhaps add `n=1, n=10` steps with an higher variance. 
- @pgrosser1 mentioned this paper https://www.jstor.org/stable/43601172?seq=18 (example of Cauchy distribution)

## Monte Carlo and Auction Pricing

- 18.2.2 - look at adding an animation to demonstrate Monte Carlo (similar to Wikipedia page)
- Add explanation for number produced by code above section 18.3
- In 18.3 explain what a European call option is (move definition towards top of section?)
- Add explanatory text at the end of Section 18.3 (to explain number better) -- perhaps relate it to parameters and see how it varies with discount factor, expected payoff goes up (for example)
- what does `{` mean after 18.4.1 equation? (independent over time sequence)
- @Jiarui-ZH (made a comment I missed here)
- Update figures to use `mystnb` metadata for Figure numbering and titling
- Backstory to `Figure above 18.4.6` (why are there two plots?)
- Change section title for `18.4.6` to `Computing the option price`
- Get more intuiting between prices and parameters

**Next Session in two weeks on 05th April**

Assignments: Please fill in here

**Comment by mbek0605** [comment]
As promised, please find minor edits for the "Distributions and Probability" lecture. 
[minor_edits_distributions_and_probabilities.pdf](https://github.com/QuantEcon/reading-group-2024/files/14711727/minor_edits_distributions_and_probabilities.pdf)

**Comment by Jiarui-ZH** [comment]
I made a note that in section 18.4.1 and 18.4.3 it would be a good idea to break down the equations in the section and talk about what are the implications behind parameters such as ξ

**Comment by HumphreyYang** [comment]
Hi @mmcky, many thanks for sharing the notes. 

Should we migrate the comments to the issue tracker in the intro series?

**Comment by mmcky** [comment]
@HumphreyYang that would be great. 👍

**Comment by HumphreyYang** [comment]
Many thanks @jstac and @mmcky,

Issues in this discussion thread have been migrated to the corresponding issue tracker.

**Reply by jstac** [reply]
Thanks @HumphreyYang , much appreciated.
